src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -285,6 +285,7 @@
   "DailyEpisodeTypeFormat": "Date ({format})",
   "Dash": "Dash",
   "Database": "Database",
+  "DatabaseMigration": "Database migration",
```

Comment:
Since we use this as a title of sorts I think `Database Migration` makes more sense. `AppData Directory` and `Startup Directory` should probably also be updated.